integration_test/multicall.test.ts

```diff
@@ -0,0 +1,88 @@
+'use strict'
+
+import { getL2Network } from '../src/lib/dataEntities/networks'
+import { providers } from 'ethers'
+import { mock, when, anything, instance, deepEqual } from 'ts-mockito'
+import { expect } from 'chai'
+
+import { MultiCaller } from '../src'
+
+describe('Multicall events', () => {
```

Comment:
Lets remove the word `events` from the title here - it's a loaded term in the EVM